pyV2DL3/tests/test_util.py

```diff
@@ -31,5 +33,101 @@ def test_getRunQuality():
     )
 
 
+def test_getGTI():
+    run_start_from_reference = 0
+    # runno 300 20 0
```

Comment:
Trivial cases are missing: all 255 and all zeros (sorry, they are trivial but good to have)

Reply:
Agreed for all 255! but all zeros - why we will use the runs having duration of 0 sec?